REVIEW_PROCESS.md

```diff
@@ -0,0 +1,63 @@
+# Introduction
+
+Articles posted to the Janelia Data Standards website should be vetted, authoritative, well-considered, and of high stylistic and technical quality. For this reason, each new article will undergo a review, using GitHub’s PR and review tooling. This process is intended to help the reviewer refine their ideas.
+
+# Review process
+
+Each new article must be approved by at least two reviewers. At least one of those reviewers must be a **full reviewer**. The other reviewer may be either a **full** or **partial** reviewer. A **full reviewer** must complete all *three* sections of the template below. A **partial reviewer** must complete *at least one* section of the template below.
+
+Reviewers are selected by the author. The author may select more than two reviewers, if desired, but only the approval of two reviewers is needed. The review *may* consist of in-line comments, but it *must* include the template. 
```

Comment:
Who decides which reviewers are full or partial? Is it the decision of the author or is it up to the reviewers to coordinate this themselves? (I would suggest that the default should be the latter, while the former could be a means to get at least one full review in the unlikely case that none of the reviewers volunteer to do it.)

Reply:
Right before Wednesday's meeting, I changed the proposal to omit the partial reviewer/full reviewer distinction. Now they are both full reviewers, but the proposal emphasizes that the review can be very short.